src/content/insertable-streams/webgpu/js/multi_video_main.js

```diff
@@ -0,0 +1,274 @@
+
```

Comment:
While prototyping some stuff based on Ben's demo page I found it could be simplified a bit to not require a vertex buffer https://paste.googleplex.com/5043473867603968 In general there are a lot of defaults and other WebGPU contruct that can be used to make the code marginally simpler (but it wouldn't make it more efficient).